src/W3C-19980720.xml

```diff
@@ -0,0 +1,65 @@
+         http://www.w3.org/Consortium/Legal/</p>
+    </copyright>
+    <optional>
+      <p>This W3C work (including software, documents, or other related items) is being provided by the copyright
+         holders under the following license. By obtaining, using and/or copying this work, you (the licensee)
+         agree that you have read, understood, and will comply with the following terms and conditions:</p>
+    </optional>
+    <body>
+      <p>Permission to use, copy, modify, and distribute this software and its documentation, with or without
+         modification, for any purpose and without fee or royalty is hereby granted, provided that you include
+         the following on ALL copies of the software and documentation or portions thereof, including
+         modifications, that you make:</p>
+      <list></list>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>The full text of this NOTICE in a location viewable to users of the redistributed or derivative work.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Any pre-existing intellectual property disclaimers, notices, or terms and conditions. If none
+             exist, a short notice of the following form (hypertext is preferred, text is permitted) should
+             be used within the body of any redistributed or derivative code: "Copyright ©
+             [$date-of-software] World Wide Web Consortium, (Massachusetts Institute of Technology,
+             Institut National de Recherche en Informatique et en Automatique, Keio University). All Rights
+             Reserved. http://www.w3.org/Consortium/Legal/"</p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>Notice of any changes or modifications to the W3C files, including the date changes were made.
+             (We recommend you provide URIs to the location from which the code is derived.)</p>
+        </li>
+      </list>
+      <p>THIS SOFTWARE AND DOCUMENTATION IS PROVIDED "AS IS," AND COPYRIGHT HOLDERS MAKE NO REPRESENTATIONS OR
+         WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO, WARRANTIES OF MERCHANTABILITY OR FITNESS
+         FOR ANY PARTICULAR PURPOSE OR THAT THE USE OF THE SOFTWARE OR DOCUMENTATION WILL NOT INFRINGE ANY
+         THIRD PARTY PATENTS, COPYRIGHTS, TRADEMARKS OR OTHER RIGHTS.</p>
+      <p>COPYRIGHT HOLDERS WILL NOT BE LIABLE FOR ANY DIRECT, INDIRECT, SPECIAL OR CONSEQUENTIAL DAMAGES ARISING
+         OUT OF ANY USE OF THE SOFTWARE OR DOCUMENTATION.</p>
+      <p>The name and trademarks of copyright holders may NOT be used in advertising or publicity pertaining to
+         the software without specific, written prior permission. Title to copyright in this software and any
+         associated documentation will at all times remain with copyright holders.</p>
+    </body>
+    <optional>
+      <p>____________________________________</p>
```

Comment:
Coming at this fresh, I would say this following should be optional. But I would also say that it's fine to leave it as-is for now and review whether to apply the &lt;optional&gt; tag more aggressively the in future.